explainer_comparison/explainer_utilities.py

```diff
@@ -1,19 +1,27 @@
 import pandas as pd
 from explainer_comparison.ExplainerFactory import ExplainerFactory
 
-def run_and_collect_explanations(factory: ExplainerFactory, X_data, explainers=None) -> pd.DataFrame:
+def run_and_collect_explanations(factory: ExplainerFactory, X_data, explainers=None, verbose=True) -> pd.DataFrame:
     results = []
-    available_explainers = ["shap", "lime"]  # Easily extendable for additional explainers
+    available_explainers = ["shap", "lime", "ebm"]#, "mimic"]  # Easily extendable for additional explainers
```

Comment:
please manage this under a config.py or similar (yml, yaml, in, etc.)

Reply:
EXPLAINERS = ["shap", "lime", "ebm"]  - added to the config.py